### PR TITLE
Multicluster - workload details page

### LINKED ISSUE #5965: Multicluster - Workloads details view

### PR DESCRIPTION
**Pull request by nrfox**
This rounds out most of the functionality for the workload details page. Traces are still not working but those will require special configuration and might best be left for a separate change.

Fixes #5965

### CODE REVIEW

**Review by hhovsepy — DISMISSED** [review]
Tested locally, checked on OCP single cluster, code is great.

File: business/proxy_status.go [hunk 1]
```diff
@@ -48,13 +47,23 @@ func xdsStatus(sent, acked string) string {
 	return "Stale"
 }
 
-func (in *ProxyStatusService) GetConfigDump(namespace, pod string) (models.EnvoyProxyDump, error) {
-	dump, err := in.k8s.GetConfigDump(namespace, pod)
+func (in *ProxyStatusService) GetConfigDump(cluster, namespace, pod string) (models.EnvoyProxyDump, error) {
```
Comment (josunect) [inline business/proxy_status.go:50]: This is just a question, should we use the Kiali Service account client here? 
Also, should this work in a primary remote deployment model? As the remote wouldn't have an Istio

**Review by hhovsepy — APPROVED** [review]

File: handlers/workloads.go [hunk 2]
```diff
@@ -105,7 +104,7 @@ func WorkloadDetails(w http.ResponseWriter, r *http.Request) {
 		includeValidations = true
 	}
 
-	var istioConfigValidations = models.IstioValidations{}
+	istioConfigValidations := models.IstioValidations{}
 	var errValidations error
```
[the remark this answers is not in the document]
Comment (hhovsepy) [reply handlers/workloads.go:108]: I will include that check in my PR https://github.com/kiali/kiali/pull/5994

**Comment by hhovsepy** [comment]
any further imporvements will come in separate PRs

**Pull request merged**